sandbox/src/DataInsertionAPI.js

```diff
@@ -0,0 +1,14 @@
+const trackingServer = "ujsl.sc.omtrdc.net";
+const reportSuite = "ujslecommerce";
+
+export function triggerAnalyticsHit({ analyticsPayload, visitorID }) {
+  const url = `https://${trackingServer}/b/ss/${reportSuite}/0?g=${window.location}&r=${document.referrer}&vid=${visitorID}&tnta=${analyticsPayload}`;
```

Comment:
This should be "mid" not "vid" for the visitor id. "mid" is for the ECID, and "vid" is for the AID

Reply:
Also, should you include "pe=tnt"?

Reply:
Thank you Jon! I'll change that. Related to `pe`:  I'm not sure if it is required, but I will double check with Ogden.